client/bank/cli/query.go

```diff
@@ -46,14 +47,18 @@ func GetAccountCmd(cdc *codec.Codec, decoder auth.AccountDecoder) *cobra.Command
 				return err
 			}
 
-			coins, err := cliCtx.ParseCoins(acc.GetCoins().String())
+			coins, err := bankcli.ConvertToMainUnit(cliCtx, acc.GetCoins())
 			if err != nil {
 				return err
 			}
 
-			acc.SetCoins(coins)
+			acc1, err := bankcli.ConvertAccountCoin(cliCtx, acc)
+			if err != nil {
+				return err
+			}
 
-			return cliCtx.PrintOutput(acc)
+			acc1.Coins = coins
+			return cliCtx.PrintOutput(acc1)
```

Comment:
If --out-put=json, expect to get the min unit coins.
Let's improve the function coins.MainUnitString() to suport any tokens.